skyve-war/src/main/java/modules/admin/UserLoginRecord/UserLoginRecord.xml

```diff
@@ -11,20 +11,25 @@
 	<bizKey expression="{userName} @ {loginDateTime}" sensitivity="personal" />
 	<attributes>
 		<text name="userName" required="true" usage="domain" sensitivity="personal">
+			<documentation>admin.userLoginRecord.userName.documentation</documentation>
```

Comment:
I don't think `documentation` supports loading via i18n? Documentation is just for developers, I don't think it needs to be internationalised.